docusaurus/docs/extensions/advanced/fix-annotations-published-extensions.md

```diff
@@ -6,15 +6,24 @@ This is due to the fact that when the first releases were done, no-one could for
 
-With Shell `3.0.0` we have introduced a mechanism that will allow developers to fix annotations on those published extension version that will handle changes on `index.yaml` that will persist over time and also the re-packaging of the `tgz` assets with the correct annotations inside. To leverage this mechanism, developers should follow this procedure:
+With Shell `3.0.0` we have introduced a mechanism that will allow developers to fix annotations on those published extension version that will handle changes on `index.yaml` that will persist over time and also the re-packaging of the `tgz` assets with the correct annotations inside (only works for Github). To leverage this mechanism, developers should follow this procedure:
 
-1) After identifying which extension version you want to fix the annotations for, go to branch of the repository where the Helm chart that serves your extension lives and edit the `Chart.yaml` in `charts/<-YOUR-EXT->/<-EXT-VERSION-TO-FIX->/Chart.yaml` and change the annotations there as desired
+**1)** Copy over the contents of the root file `package.json` to have access to the yarn command alias `publish-pkgs`. You'll need to create the file on the branch where the Helm chart that serves your extension lives in
 
-2) Once the annotations changes are done, on the root folder of the skeleton app that contains the extension just run the following command:
+**2)** After identifying which extension version you want to fix the annotations for, go to branch of the repository where the Helm chart that serves your extension lives (for the examples we [provide](./../extensions-getting-started.md#creating-a-release) that branch should be `gh-pages`) and edit the `Chart.yaml` in `charts/<-YOUR-EXT->/<-EXT-VERSION-TO-FIX->/Chart.yaml` and change the annotations there as desired
+
+**3)** On the same branch, create on the root a file called `package.json`, with the contents copied from step 1)
+
+**4)** Install node modules by running `yarn install`
+
+
+**5)** One the root of your branch, just run the following command:
 ```js
-yarn publish-pkgs -e -g
+yarn publish-pkgs -e -s <-GITHUB-ORG->/<-GITHUB-REPO->
 ```
 
+In the case of the [Elemental repo](https://github.com/rancher/elemental-ui), the `GITHUB-ORG` would be `rancher` and `GITHUB-REPO` would be `elemental`
+
 This will update the `index.yaml` of your Helm chart, propagating those annotation changes and also re-package the `tgz` file under `assets`.
 
-3) After the procedure is finished, just create a PR for your repository and merge your changes. The annotations that were changed in those published version should now be according to your needs
\ No newline at end of file
+**6)** After the procedure is finished, just create a PR for your repository and merge your changes. The annotations that were changed in those published version should now be according to your needs. Careful to only commit the needed changes and not your new `package.json` or the `/node_modules` folder
```

Comment:
```suggestion
**6)** Once the procedure is complete, remove the `package.json`, `yarn.lock`, and `/node_modules` folder from your branch to avoid committing them.

**7)** Create a pull request for your repository and merge your changes. The annotations for the published version should now reflect your updates.
```